db/migrate/20210409091635_create_active_admin_comments.rb

```diff
@@ -0,0 +1,16 @@
+class CreateActiveAdminComments < ActiveRecord::Migration[6.1]
+  def self.up
+    create_table :active_admin_comments do |t|
+      t.string :namespace
+      t.text   :body
+      t.references :resource, polymorphic: true
+      t.references :author, polymorphic: true
+      t.timestamps
+    end
+    add_index :active_admin_comments, [:namespace]
+  end
+
+  def self.down
+    drop_table :active_admin_comments
+  end
```

Comment:
why this type of name of a table?

Reply:
it was built in when, generated Active Admin. If I will change it,  will face issues where ever it is used.